models/image.py

```diff
@@ -776,17 +789,17 @@ def from_images(cls, images):
         output.end_mjd = max([image.end_mjd for image in images])  # exposure ends are not necessarily sorted
 
         # TODO: what about the header? should we combine them somehow?
```

Comment:
Maybe add a header HISTORY or COMMENT that lists all the images that were combined?

Or, a series of keywords COMBIMn, where n is an increasing integer (and corresponds directly to the indexes of the combined images), and the value of each is the image ID.  (With a comment indicating that.)

Reply:
I've made an issue about this (#172 ) we should discuss.